pkg/constants/constants.go

```diff
@@ -53,4 +53,6 @@ const (
 
 	DefaultConfigFileName = ".avalanche-cli"
 	DefaultConfigFileType = "json"
+
+	DefaultPluginDir = "plugins"
```

Comment:
really this is not going to be the plugin dir used by avalanchego, probably the name DefaultPluginDir is misleaving

Reply:
this is a good point, I moved the referenced plugin dir to `/tmp`